server.js

```diff
@@ -1,22 +1,63 @@
+if (process.env.NODE_ENV !== 'production') {
+  require('dotenv').config()
+};
+
 // Dependences
 const express = require("express");
+
+const session = require('express-session');
+const passport = require('passport');
+const flash = require('express-flash');
+const db = require('./models');
+const sequelize = require('sequelize');
+
 const messaging = require("./controllers/messaging/messagingServer");
 
+
 // Express config
 const app = express();
-const PORT = process.env.PORT || 3001;
+const PORT = process.env.PORT || 7001;
```

Comment:
Any reason with changing the port number. Please revert this as the rest of us have been using port 3001 since boilerplate.

Reply:
just always used 7001 >.< ill change it